### PR TITLE
add DQM Validation plots for MSSM Hbb Trigger Paths.

### PR DESCRIPTION
**Pull request by heppye**
add DQM Validation plots for MSSM Hbb Trigger Paths.

### CODE REVIEW

**Comment by heppye** [comment]
a rebasing version for the original PR https://github.com/cms-sw/cmssw/pull/10550.

**Comment by davidlange6** [comment]
@cmsbuild, please test

**Comment by heppye** [comment]
@davidlange6, thanks for your quick response. @deguio , could you please check the PR again?

File: HLTriggerOffline/Higgs/python/hltHiggsPostProcessors_cff.py [hunk 1]
```diff
@@ -320,6 +320,24 @@ def get_reco_strings(strings):
 hltHiggsPostWHToENuBB.subDirs = ['HLT/Higgs/WHToENuBB']
 hltHiggsPostWHToENuBB.efficiencyProfile = efficiency_strings_WHToENuBB
 
+#Specific plots for MSSMHbb
+#Jet plots
+NMinOneCuts = _config.MSSMHbb.NminOneCuts
```
Comment (deguio) [inline HLTriggerOffline/Higgs/python/hltHiggsPostProcessors_cff.py]: I guess this is a typo.
NMinOneCuts should be NminOneCuts

**Comment by deguio** [comment]
please see my comment. in case the same fix should be submitted for 75 and 76 as well.
thanks,
F.

**Comment by heppye** [comment]
@deguio, fixing done. PR for CMSSW_7_6_X https://github.com/cms-sw/cmssw/pull/11235 and PR for CMSSW_7_5_X https://github.com/cms-sw/cmssw/pull/11234 have been made as well. I am sorry for the mistakes.

**Comment by deguio** [comment]
please test

**Comment by deguio** [comment]
+1
provided it passes the tests

**Comment by davidlange6** [comment]
+1

**Pull request merged**